src/components/projects/CreateProject.jsx

```diff
@@ -323,14 +325,27 @@ const SubmissionSection = ({ classes, history }) => {
               }}
               id="goalToken"
               name="goalToken"
+              select
               label="Select Token"
               placeholder="Select Token"
               margin="normal"
               variant="outlined"
               onChange={handleChange}
               onBlur={handleBlur}
-              value={values.token || ''}
-            />
+              value={values.goalToken || ''}
+            >
+              {currencies.map((option) => (
+                <MenuItem style={{display: 'flex', alignItems: 'center'}} key={option.value} value={option.value}>
+                  <div style={{display: 'flex', alignItems: 'center'}}>
+                    {option.icon || <img
+                      src={option.img || `${TOKEN_ICON_API}/${option.value}.png`}
+                      style={{width: option.width, marginRight: '3%'}}
+                    />}
+                    {option.label}
+                  </div>
+                </MenuItem>
+              ))}
+            </TextField>
```

Comment:
Could we create a component that can be reused for both this and the management page?

Reply:
yes, but they are different components, so how do you see this components being called with what props?

```
<CurrencySelect 
              className={classes.textField}
              InputProps={{
                classes: {
                  input: classes.textInput
                }
              }}
              id="goalToken"
              name="goalToken"
              label="Select Token"
              onChange={handleChange}
              onBlur={handleBlur}
              value={values.goalToken}
              showBalance={false}
              toggleAllowance={false}
/>
```

Reply:
Yeah that looks good. Basically, most/all of the props you already give to `TextField` can be sent instead to the custom component